website/docs/api/plugins/plugin-content-blog.mdx

```diff
@@ -130,6 +130,13 @@ type CreateFeedItemsFn = (params: {
 }) => Promise<BlogFeedItem[]>;
 ```
 
+#### `SortBlogPostsFn` {#SortBlogPostsFn}
+
+```ts
+// TODO document this type
+((args: {blogPosts: BlogPost[]}) => void | BlogPost[])
```

Comment:
``` suggestion
type SortBlogPostsFn = (params: {blogPosts: BlogPost[]}) => void | BlogPost[]
```

Follow the declaration convention of type params above: give a name to type and use `params` instead of `args` (also in code if possible)